css-logical-1/Overview.bs

```diff
@@ -220,6 +220,22 @@ Flow-Relative Box Model Properties</h2>
   depends on the element’s computed values of
   'writing-mode', 'direction', and 'text-orientation'.
 
+  Each set of flow-relative properties and their parallel physical
+  properties form a <dfn export>logical property group</dfn>.
+
+  Each property in a <a>logical property group</a> has
+  a <dfn export>logical kind</dfn>. Flow-relative properties'
+  <a>logical kind</a> is "flow-relative", and physical properties'
+  <a>logical kind</a> is "physical".
+
+  Note: For example, 'block-size', 'inline-size', 'width', and 'height'
+  form a <a>logical property group</a>, while 'margin-block-start',
+  'margin-block-end', 'margin-inline-start', 'margin-inline-end',
+  'margin-top', 'margin-right', 'margin-bottom', and 'margin-left'
+  form another.
```

Comment:
What about shorthand properties like `margin-block`?

Reply:
Regarding the proposed use in #2924, it doesn't matter. Shorthand properties are expanded to longhands during parsing.

Reply:
Yes, but this should be properly defined independently of #2924. "Each set of flow-relative properties" seems to include shorthands, but they are not in the example. You can say "Each set of flow-relative longhand properties"

Reply:
Changed that according to your suggestion. Thanks.